components/config/src/config/mod.rs

```diff
@@ -312,35 +454,63 @@ impl Default for Config {
     fn default() -> Config {
         Config {
             base_url: DEFAULT_BASE_URL.to_string(),
-            title: None,
-            description: None,
             theme: None,
             highlight_code: false,
             highlight_theme: "base16-ocean-dark".to_string(),
-            default_language: "en".to_string(),
-            languages: Vec::new(),
-            generate_feed: false,
-            feed_limit: None,
+            lang: langid!("en"),
```

Comment:
do we need that macro? Is the macro-less approach harder to use?

Reply:
We could have it be parsed from a str. Parsing and using an `unwrap()` in a constructor seems a bit dodgy to me.